includes/Modules/Analytics.php

```diff
@@ -919,7 +919,7 @@ function ( $dimension_def ) {
 					// When using multiple date ranges, it changes the structure of the response,
 					// where each date range becomes an item in a list.
 					if ( ! empty( $data['multiDateRange'] ) ) {
-						$date_ranges[] = $this->parse_date_range( $date_range, 1, 1, true );
+						$date_ranges[] = $this->parse_date_range( $date_range, 1, 1, true, true );
```

Comment:
The signature of this method is growing rather cryptic. It's probably beyond the scope of this issue to change, but I think it would be much more human-friendly if it took an array of arguments rather than a list of positional numbers and booleans. Can we open a new issue to address this?

Reply:
Agreed, I will open a separate issue for this.

Reply:
Created issue: https://github.com/google/site-kit-wp/issues/1877